ui-modules/blueprint-composer/app/components/util/d3-blueprint.js

```diff
@@ -771,50 +781,35 @@ export function D3Blueprint(container, options) {
         return node;
     }
 
+    /**
+     * Draws new relationships as arcs and labels. Updates existing ones.
+     */
     function drawRelationships() {
+
         showRelationships();
 
-        let relationData = _relationGroup.selectAll('.relation')
-            .data(_d3DataHolder.visible.relationships, (d)=>(d.source._id + '_related_to_' + d.target._id));
+        const getPathId = ({ source, target }) => `${source._id}-${target._id}`;
+        const PATH_SELECTOR = 'relation-arc';
+        const LABEL_SELECTOR = 'relation-label';
```

Comment:
[MINOR] The convention up until now is to put constant into the `_configHolder`

Reply:
Addressed in 94485d2c28c48a6aadb92f167fddc284eab02e8f.